tools/api_proto_breaking_change_detector/detector_test.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+import unittest
+
+from detector import BufWrapper, ChangeDetectorInitializeError
+
+
+class BufTests(unittest.TestCase):
+
+    def run_buf_test(self, testname, is_breaking, expects_changes, additional_args=None):
+        tests_path = Path(
+            Path(__file__).absolute().parent.parent, "testdata",
+            "api_proto_breaking_change_detector", "breaking" if is_breaking else "allowed")
+
+        current = Path(tests_path, f"{testname}_current")
+        changed = Path(tests_path, f"{testname}_next")
+
+        breaking_response = BufWrapper.is_breaking(current, changed, additional_args)
+        self.assertEqual(breaking_response, is_breaking)
+
+        lock_file_changed_response = BufWrapper.lock_file_changed(current, changed, additional_args)
+        self.assertEqual(lock_file_changed_response, expects_changes)
+
+
+class TestBreakingChanges(BufTests):
+
+    def test_change_field_id(self):
+        self.run_buf_test(
+            self.test_change_field_id.__name__, is_breaking=True, expects_changes=False)
```

Comment:
Seems that all breaking tests are passed with the same parameters. A wrapper function that sets these parameters will make this more readable, as this is the typical path. (similarly for the allowed tests)